client/public/assets/css/style.css

```diff
@@ -34,19 +34,21 @@ nav .brand-logo {
     width: 150px;
 }
 
-#content {
-    position: absolute;
+#canvasWrapper {
+    position: relative;
```

Comment:
Does this need to be position relative? What issue does this fix? Why is `34px` the magical offset?